puns.json

```diff
@@ -130,5 +130,9 @@
   {
     "pun": "Q: Why doesn't the developer use Git?",
     "punchline": "A: Because they're afraid to commit!"
+  },
+  {
+    "pun": "Q: Why does a front end developer eat alone?",
+    "punchline": "A: Because he doesn't know how to join the tables!"
```

Comment:
I try to keep these gender-neutral. Can you please change it to "Because they don't know how to join the tables!"?

Reply:
Well, it's a tricky one as I'm not sure if we should use "they" because then it makes it less funny (they could sit together, right?).

```
Q: Why do the front end developers eat alone?
A: Because they don't know how to join the tables!
```

I get your point though 🤔

Reply:
English has a singular they pronoun, so it doesn't have to be plural "front end developers".  It can still be `front end developer`.
https://en.wikipedia.org/wiki/Singular_they

Reply:
Well, fair enough! :)